algosdk/abi/contract.py

```diff
@@ -28,19 +29,19 @@ def __eq__(self, o) -> bool:
         )
 
     @staticmethod
-    def from_json(resp):
+    def from_json(resp: Union[str, bytes, bytearray]) -> "Contract":
```

Comment:
I agree with this approach for our SDK, rather than self-referencing `Contract` (with a `from __future__ import annotations` above). That's because the [future may never arrive](https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class).